z4yed/react-dashboard
# Setting button added at right-bottom

# Steps to verify
1. Run the project by `npm run start`
2. Visit [http://localhost:3000/](http://localhost:3000/)
3. You'll see a setting icon positioned at the right bottom. 
  -Tooltip message is 'Settings`. Hover to see.
